marklogic-data-hub/src/test/resources/test-projects/provenance-test/flows/inline.flow.json

```diff
@@ -29,6 +29,7 @@
           "name": "SimpleCustomerMapping",
           "version": 1
         },
+        "provenanceGranularityLevel": "fine",
```

Comment:
I think it's worth adding an assertion to GetRecordTest (it's tangential to the test, but I think it's fine to stuff it in there) verifying that there's no error in stepOutput, and the test should verify that prov is set to "fine" in the step config as well. The assertion messages can state that we expect the step to not fail, even though fine is not supported for a mapping step.